pkgs/sdk-pkg/src/genie/libs/sdk/apis/iosxr/platform/get.py

```diff
@@ -142,9 +142,13 @@ def get_available_space(device, directory='', output=None):
         return None
 
     free_bytes = Dq(dir_output).get_values(key='total_free_bytes')
-    bytes_free = int(re.search(r'\d+', free_bytes[0]).group(0))
-
-    if bytes_free:
+    reg = re.search(r'(\d+)\s(\S+)', free_bytes[0])
```

Comment:
Do instances exist where `kbytes` wouldn't exist? IE, where this regex wouldn't match and cause the API to fail?

Reply:
I've tried on recent versions no both ncs5k and asr9k, the output is similar. But in theory I guess it can happen, yes. 
What would you think about `r"(\d+)\s*(\S+)*"` ? Then I can check if the `str` has been captured and do the conversion (default will be to keep the numeric value as is).

Reply:
That would work well. That or `r"(\d+)\s*(\S+)?"` which is effectively the same thing I think

Reply:
Ok, I'll make the change asap. Thx !